POC for multi-session app (#954)
inrupt/solid-client-authn-js: packages/node/example/multiSession/README.md

> # Demo: a server NodeJS app managing sessions

Reviewer: ```suggestion
# Demo: a server-side NodeJS application managing multiple sessions
```

This demo shows how you can run a NodeJS server with multiple sessions in parallel.
Check out the code in `src/serverSideApp.ts` for more details.

Note that until [this issue](https://github.com/solid/node-solid-server/issues/1533)
is resolved, this will not work against NSS. This demo is only intended to work against a compliant

> OIDC implementation, such as Inrupt's ESS (e.g. deployed at `https://pod.inrupt.com`).

Reviewer: Our writing guidelines state 'i.e.' and 'e.g.' should be followed by a comma. I'd never of heard of it until Kevin brought it up recently (and it still seems weird to me), but apparently it is more correct English.
```suggestion
OIDC implementation, such as Inrupt's ESS (e.g., deployed at `https://pod.inrupt.com`).
```

Author: Oh right, yeah I was told that rule in the past

## Installing the app

> To properly install this demo app, first run `npm ci` at the root of the repository, which will trigger
> the build of the library. Then, run `npm ci` again, this time in
> `packages/node/example/multiSession/`, and then `npm run build`. You should be all set. If you change the

Reviewer: ```suggestion
To install this demo app, first run `npm ci` at the root of the repository, which will trigger
the build of the libraries our app depends on. Then, run `npm ci` again, this time in
`packages/node/example/multiSession/`, and then `npm run build`. You should be all set. If you change the
```
